NEMO/templates/calendar/reservation_event_feed.html

```diff
@@ -7,6 +7,12 @@
 		{% elif personal_schedule and x.area %}
 			"title": "{{ x.title|default:x.reservation_item.name|escapejs }}",
 			"color": "#84CD84",
+		{% elif all_tools and x.tool %}
+			"title": "{{ x.reservation_item.name|escapejs }}\n{{ x.title|escapejs }}",
+			"color": "{{ x.reservation_item.tool_calendar_color|default:'#33ad33'|escapejs }}",
+		{% elif all_tools and x.area %}
+			"title": "{{ x.reservation_item.name|escapejs }}\n{{ x.title|escapejs }}",
+			"color": "#84CD84",
```

Comment:
I think this part is irrelevant since you should not see any area reservations when all_tools is selected